cscs-checks/compile/libsci_resolve.py

```diff
@@ -18,7 +18,7 @@ class LibSciResolveBaseTest(rfm.CompileOnlyRegressionTest):
 
```

Comment:
I think that just setting the executable here should do the trick. No need to replace the `executable` by the `name` in the hook below
```suggestion
    executable = 'libsci.x'
```